tests/test.fish

```diff
@@ -38,11 +46,11 @@ if set -q files_to_test[1]
     test "$FISH_FORCE_COLOR" = 1
     and set force_color --force-color
 
-    $python -S ../littlecheck.py \
+    $python -S $scriptdir/littlecheck.py \
         --progress $force_color \
-        -s fish=../test/root/bin/fish \
-        -s fish_test_helper=../test/root/bin/fish_test_helper \
-        -s filter-control-sequences='../test/root/bin/fish ../tests/filter-control-sequences.fish' \
+        -s fish=$FISHDIR/fish \
+        -s fish_test_helper=$fish_test_helper \
+        -s filter-control-sequences="$FISHDIR/fish $scriptdir/filter-control-sequences.fish" \
```

Comment:
This will fail if $FISHDIR or $scriptdir include spaces.

I don't know how to solve it yet.